src/main/java/seedu/mtracker/instrument/InstrumentManager.java

```diff
@@ -5,7 +5,7 @@
 public class InstrumentManager {
 
     private static InstrumentManager instrumentManager;
-    private final ArrayList<Instrument> instruments;
+    private static ArrayList<Instrument> instruments;

```

Comment:
Is there a reason why this was changed to static? It should an instance member and not a class member

Reply:
Final are for variables that are assigned only once, not necessarily for constants only, so the naming still stays